clouddriver-google-common/src/main/groovy/com/netflix/spinnaker/clouddriver/googlecommon/batch/GoogleBatchRequest.java

```diff
@@ -41,6 +41,7 @@
 public class GoogleBatchRequest {
 
   private static final int MAX_BATCH_SIZE = 100; // Platform specified max to not overwhelm batch backends.
+  private static final int DEFAULT_EXECUTE_TIMEOUT_MINUTES = 10;
```

Comment:
What's the relationship between `DEFAULT_READ_TIMEOUT_MILLIS`, and the total execution time here?

Reply:
Looks like `DEFAULT_READ_TIMEOUT_MILLIS` deals with the open connection in the batch and `DEFAULT_EXECUTE_TIMEOUT_MINUTES` is for threadpool shutdown as it is now. Should those two be the same?

Reply:
My first guess was this should be: `DEFAULT_EXECUTE_TIMEOUT_MINUTES = DEFAULT_CONNECT_TIMEOUT_MILLIS + DEFAULT_READ_TIMEOUT_MILLIS + small_constant`, but I wasn't sure if we do multiple sequential reads, and if the fact that it now looks like we allow for ~4 of those assuming the chosen constants were intentional.